front/app/api/phase_permissions/keys.ts

```diff
@@ -0,0 +1,11 @@
+import { PhasePermissionsProps } from './usePhasePermissions';
+
+const phasePermissionKeys = {
+  all: () => [{ type: 'permission', variant: 'phase' }],
+  lists: () => [{ ...phasePermissionKeys.all()[0], operation: 'list' }],
+  list: ({ phaseId }: PhasePermissionsProps) => [
+    { ...phasePermissionKeys.lists()[0], phaseId },
+  ],
+} as const;
```

Comment:
Still need to do `satisfies QueryKeys` here

Reply:
Ah shoot - yep I forgot that one. Probably the project permissions one will also need it. Fixing it now! Thanks!!